static/src/javascripts/projects/common/modules/experiments/tests/giraffe-header.js

```diff
@@ -0,0 +1,72 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svgs',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/giraffe-header.html'
+], function (bean,
+             qwery,
+             $,
+             template,
+             svgs,
+             fastdom,
+             mediator,
+             giraffeHeader
+) {
+    return function () {
+
+        this.id = 'GiraffeHeader20160802';
+        this.start = '2016-08-02';
+        this.expiry = '2016-08-22';
+        this.author = 'Mark Butler';
+        this.description = 'Add a header allowing readers to contribute money.';
+        this.showForSensitive = false;
+        this.audience = 0.10;
+        this.audienceOffset = 0;
+        this.successMeasure = 'Determine if header is a good mechanism for driving contributions.';
+        this.audienceCriteria = 'All users';
+        this.dataLinkNames = '';
+        this.idealOutcome = '';
+        this.canRun = function () {
+            var pageObj = window.guardian.config.page;
+            return !(pageObj.isSensitive || pageObj.isLiveBlog || pageObj.isAdvertisementFeature) && pageObj.edition === 'UK';
```

Comment:
https://github.com/guardian/frontend/blob/3194e47/static/src/javascripts/projects/common/modules/experiments/ab.js#L111-L113 seems to indicate that `showForSensitive` is keyed off `shouldHideAdverts`, which sounds reasonable - I'll take out the `pageObj.isSensitive` clause.